src/ApiConnectors/BaseApiConnector.php

```diff
@@ -161,4 +166,70 @@ protected function getFinderService(): FinderService
     {
         return $this->connection->getAuthenticatedClient(Services::FINDER());
     }
-}
+
+    /**
+     * Convert options array to an ArrayOfString which is accepted by Twinfield.
+     *
+     * @param array $options
+     * @param array|null $forcedOptions
+     * @return array
+     * @throws Exception
+     */
+    public function convertOptionsToArrayOfString(array $options, array $forcedOptions = null): array {
+        if (isset($options['ArrayOfString'])) {
+            return $options;
+        } else {
```

Comment:
These is no need to have an `else` if the if body has a `return` statement, can you change this to use less indenting?

Reply:
Done